litgpt/finetune/adapter_v2.py

```diff
@@ -33,12 +33,6 @@
     save_hyperparameters,
 )
 
-# support running without installing as a package
```

Comment:
If we still want the scripts to be executable without the package install, we will need to keep these (and re-add them to the other scripts that had them removed).

Reply:
Readded. I think we'll want to remove them after we have the cli